assets/src/dashboard/components/typeaheadOptions/index.js

```diff
@@ -118,19 +126,63 @@ const TypeaheadOptions = ({ isOpen, items, maxItemsVisible = 5, onSelect }) => {
   }, [hoveredIndex, items, onSelect, isOpen]);
 
   useEffect(() => {
-    if (listRef.current) {
-      listRef.current.scrollToItem(0);
+    if (!listRef.current) {
+      return () => {};
     }
-    setHoveredIndex(0);
-  }, [items]);
+
+    listRef.current.children[0].focus();
```

Comment:
Should we also check to see here if `children[0]` exists? I know we are seeing if `current` does.